README.md

```diff
@@ -1,6 +1,8 @@
 The leaderboard website displays scores submitted by different players.
 
-Screenshot (222)
+## Project Design:
+
+![screenshot](Screenshot.png)
 
```

Comment:
- [ ] README file should have a title of the project so that anyone can understand your project. Also, include the live link or remove the live link section to make the README professional.